### ISSUE TITLE
Multiple configuration files

### ISSUE THREAD

**Issue by alvarotrigo** [markstory/asset_compress#264]
It would be nice to be able to define different configuration files.
In my case, I have a website and then an application inside it. It would be ideal to be able to separate both sites, as they don't have any single javascript file in common.

**Comment by markstory** [comment]
Does the `--config` option not work?

**Comment by alvarotrigo** [comment]
I haven't found any reference to that option in the wiki.

**Comment by markstory** [comment]
Its in the help for the cli tool.

I'll add it to the wiki as well.

**Comment by alvarotrigo** [comment]
Right. I see it now. But yeah, it would be nice having it in the wiki as well.

**Comment by alvarotrigo** [comment]
I've seen that if I have two config files and none of them has the default name `asset_compress.ini`, then an error appears:

> Configuration file "C:\inetpub\wwwroot\cakephp\app\Config\asset_compress.ini" was not found.

Does it means I have to create an empty file with the name `asset_compress.ini`? Or is there a way to configure that as well?

**Comment by markstory** [comment]
I updated the wiki for the command line flag.

Yes, the plugin assumes and expects you have a config file with the default name. I don't think anyone has earnestly wanted to use multiple config files before.

**Comment by alvarotrigo** [comment]
> Yes, the plugin assumes and expects you have a config file with the default name. I don't think anyone has earnestly wanted to use multiple config files before.

In that case, does it matter at all which of the config files is as the default one? Will it have any consequences on the page?

**Comment by markstory** [comment]
Telling the various parts of the plugin which is 'default' is the only challenging thing I think.
